pcweb/pages/docs/custom_components.py

```diff
@@ -120,6 +119,15 @@ def pypi_summary(summary: str) -> rx.Component:
     )
 
 
+def pip_install_command_copy_button(package_name: str) -> rx.Component:
+    return rx.link(
+        info_icon(
+            tag="clipboard-copy",
+            on_click=rx.set_clipboard(f"pip install {package_name}"),
```

Comment:
is this not a valid var operation? the page doesn't render

Reply:
@picklelo

Reply:
I think it's a bug, I changed this to be a code block instead